notify/notify.go

```diff
@@ -624,8 +623,11 @@ func (r RetryStage) Exec(ctx context.Context, l log.Logger, alerts ...*types.Ale
 
 		select {
 		case <-tick.C:
-			if retry, err := r.integration.Notify(ctx, alerts...); err != nil {
-				numFailedNotifications.WithLabelValues(r.integration.name).Inc()
+			now := time.Now()
+			retry, err := r.integration.Notify(ctx, alerts...)
+			notificationLatencySeconds.WithLabelValues(r.integration.name).Observe(time.Since(now).Seconds())
+			if err != nil {
+				numFailedNotifications.WithLabelValues(r.integration.name, r.groupName).Inc()
```

Comment:
The definition of `numFailedNotifications` hasn't been updated with the new label.